src/components/study-container.js

```diff
@@ -510,7 +508,11 @@ export function StudyContainer({ view, onChangeTab }) {
                 // updating data related to impacted substations
                 if (substationsIds?.length > 0) {
                     console.info('Reload network equipments');
-                    network.reloadImpactedSubstationsEquipments(substationsIds);
+                    network.reloadImpactedSubstationsEquipments(
+                        studyUuid,
+                        currentNode,
```

Comment:
Use `currentNodeRef.current` to avoid to proc this effect when currentNode changes ?
`currentNodeRef.current` is updated in another Effect, maybe we should order our effects then.

Reply:
Yes it limits the number of times this effect is executed , I changed it.
But I don't think we need to change the order